registry/registry_internals.h

```diff
@@ -12,28 +12,28 @@
 
 // ----------------------------------------------------------------------------
 // COMMON structures
-
+#include <stdint.h>
```

Comment:
Please remove this line. It is included from `libnetdata.h` that already has:
```
#if defined(HAVE_INTTYPES_H)
#include <inttypes.h>
#elif defined(HAVE_STDINT_H)
#include <stdint.h>
#endif
```

Reply:
Ok